cropper/app/lib/Crops.scala

```diff
@@ -31,15 +31,23 @@ class Crops(config: CropperConfig, store: CropStore, imageOperations: ImageOpera
     s"${source.id}/${Crop.getCropId(bounds)}/${masterString}$outputWidth${fileType.fileExtension}"
   }
 
-  def createMasterCrop(apiImage: SourceImage, sourceFile: File, crop: Crop, mediaType: MimeType, colourModel: Option[String],
-                      colourType: String)(implicit requestContext: RequestLoggingContext): Future[MasterCrop] = {
+  def createMasterCrop(
+    apiImage: SourceImage,
+    sourceFile: File,
+    crop: Crop,
+    mediaType: MimeType,
+    colourModel: Option[String],
+  )(implicit requestContext: RequestLoggingContext): Future[MasterCrop] = {
 
     val source   = crop.specification
     val metadata = apiImage.metadata
     val iccColourSpace = FileMetadataHelper.normalisedIccColourSpace(apiImage.fileMetadata)
 
     for {
-      strip <- imageOperations.cropImage(sourceFile, apiImage.source.mimeType, source.bounds, masterCropQuality, config.tempDir, iccColourSpace, colourModel, mediaType)
+      strip <- imageOperations.cropImage(
+        sourceFile, apiImage.source.mimeType, source.bounds, masterCropQuality, config.tempDir,
+        iccColourSpace, colourModel, mediaType, isTransformedFromSource = false
```

Comment:
Why is `isTransformedFromSource` `false` here?

Reply:
Answer from @andrew-nowak – we're dealing with the original file (see all the references to `source`), so it won't have been through the image pipeline.